Makefile

```diff
@@ -0,0 +1,16 @@
+all: install build deploy
+
+.PHONY+=install
+install:
+	npm ci
+
+.PHONY+=build
+build: install
+	npm run build
+
+local: build
```

Comment:
should depend on install instead of build.

Reply:
done